indexclient/client.py

```diff
@@ -280,16 +285,21 @@ def create_alias(
         resp = self._put(url, headers=headers, data=data, auth=self.auth)
         return resp.json()
 
-    def get_latest_version(self, did, skip_null_versions=False):
+    def get_latest_version(self, did, skip_null_versions=False, skip_deleted_versions=True):
         """
+        Get the latest version given did
         Args:
             did (str): document id of an existing entry whose latest version is requested
             skip_null_versions (bool): if True, exclude entries without a version
+            skip_deleted_versions (bool): if True, exclude entries marked as deleted in metadata
         Returns:
             Document: latest version of the entry
         """
 
-        params = {"has_version": "true" if skip_null_versions else "false"}
+        params = {
+            "has_version": "true" if skip_null_versions else "false",
```

Comment:
Hmmm we do this string conversion a lot. Maybe we should make a utils type function.

Reply:
If we are worried about the null you can always "cast" it to bool. But is None a valid input? If not then we prob shouldn't worry about it.
```suggestion
            "has_version": json.dumps(bool(skip_null_version))
```

Reply:
Good call! For these particular instances None would not be valid. Thank you for the recommend, I'm going to implement and commit those today.